### ISSUE TITLE
M1/Monterey

### ISSUE THREAD

**Issue by willgarland** [danielecook/seq-utilities#1]
Hi, is there any chance you will be updating this for newer M1 Macs with Monterey? Being able to reverse complement a sequence in an instant is such an invaluable tool.

**Comment by danielecook** [comment]
What happens when you try to run the workflow?

Unfortunately I do not have an M1 mac so i'm not able to test currently.

**Comment by willgarland** [comment]
As python 2 is not included in Apple's pre-installed scripting languages, it doesn't run the workflow and will just use the input for a web search. 
I've tried installing Python via homebrew but that hasn't solved it yet. 
<img width="851" alt="image" src="https://user-images.githubusercontent.com/75995523/163679067-27d1c7f6-1f54-4504-8e7e-3225c4116205.png">

**Comment by danielecook** [comment]
Here is a quick workaround:

https://www.alfredapp.com/help/kb/python-2-monterey/

Unfortunately, it looks like the workflow library that this workflow uses is not python3 compatible - making a quick fix a little tricky.

**Comment by willgarland** [comment]
Yeah I tried that workaround and changed the script language to opt/homebrew/bin/python but it's still not working
